about.html

```diff
@@ -0,0 +1,150 @@
+                <li><a href="#logo">Mission</a></li>
+                <li><a href="#past-events">Events</a></li>
+                <li><a href="#partners">Partners</a></li>
+                <li><a href="#">Blog</a></li>
+            </ul>    
+        </div>
+    </header>
+
+    <main>
+        <section id="headline">
+            <div class="wrapper headline-content t-center">
+                <h1>A mission to share a common Goal</h1>
+                <div class="headline-about-description">
+                    <p>Our mission: to help people learn to code for free. We accomplish this by creating thousands of videos, articles, and interactive coding lessons - all freely available to the public. We also have thousands of freeCodeCamp study groups around the world.
+
+                        Donations to freeCodeCamp go toward our education initiatives, and help pay for servers, services, and staff.</p>
+                </div>
+                <div class="headline-about-footer">
+                    <h2>Please get in touch with us for more information at:</h2>
+                    <h3>ahmed.hasan.rony@gmail.com</h3>
+                </div>
+            </div>
+        </section>
+
+        <section id="logo">
+            <div class="logo-title">
+                <h2>DrSight - Your Desire Our Priority</h2>
+                <hr>
+            </div>
+            <div class="logo-description">
+                <p class="t-center">
+                    drsight is a community of millions of people around the world who are learning to code together. You can learn to code with free online courses, programming projects, and interview preparation for developer jobs. We're a 501(c)(3) public charity.
+                </p>
+            </div>
+            <div class="logo-content">
+                <img src="images/logo.jpg" alt="logo">
+            </div>
+        </section>
+
+        <section id="past-events">
+            <div class="events-title">
+                <h2>Past Global Events</h2>
+                <hr>
+                <p class="t-center">Experience our past two events in photos; locations New York, New York, and Jerusalem, Israel.</p>
+            </div>
+            <div class="events-container">
+                <div class="event-card">
+                    <h2>2022</h2>
+                    <h3>New York, New York</h3>
+                </div>
+                <div class="event-card">
+                    <h2>2022</h2>
+                    <h3>Jerusalem, Israel</h3>
+                </div>
+            </div>
+        </section>
+
+        <section id="partners">
+            <div class="partner-title">
+                <h2>Partner</h2>
+                <hr>
+            </div>
+
+            <div class="partner-card">
+                <div class="partner-logo">
+                    <img src="images/partner1.png" alt="partner 1">
+                </div>
+                <div class="partner-logo">
```

Comment:
- [ ] Kindly make sure to remove the partner section on the about page as it's only present on the home page.

Reply:
🔴 removed from the DOM 😄